src/navigations/HomeBottomTabs.js

```diff
@@ -33,6 +34,7 @@ function HomeBottomTabs() {
   const {signedChannelUnreadCount, anonymousChannelUnreadCount} = useRootChannelListHook();
   const navigation = useNavigation();
   const {isLoadingFetchingChannelDetail} = usePushNotificationHook();
+  const {udpateAppBadgeWith} = useAppBadgeHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a typo in the function name `udpateAppBadgeWith`. It should probably be `updateAppBadgeWith`. Please correct this to avoid confusion and potential bugs.
```diff
- const {udpateAppBadgeWith} = useAppBadgeHook();
+ const {updateAppBadgeWith} = useAppBadgeHook();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->